features/step_definitions/api/application_plans_steps.rb

```diff
@@ -10,6 +10,23 @@
   assert_equal @plan, default_service.reload.default_application_plan
 end
 
+When "the default application plan is set" do
+  plan = FactoryBot.create(:application_plan, issuer: default_service)
+  default_service.default_application_plan = plan
+  default_service.save!
+end
+
+When "an admin unsets the default application plan" do
+  visit admin_service_application_plans_path(default_service)
+  select_default_plan_by_name "No plan selected"
+end
+
+Then "the service will not have the default plan set" do
+  visit admin_service_application_plans_path(default_service)
+  sleep 5
+  assert_equal('No plan selected', find('.pf-c-select input').value)
```

Comment:
```suggestion
  assert find('#default_plan_card').has_text?('No plan selected')
```
Or
```suggestion
  assert has_css?('#default_plan_card .pf-c-select input, text: 'No plan selected')
```

Reply:
Why?

Actually, neither of them work properly, I guess `text` is for text inside the elements, not the element attributes (i.e. `value` attribute of the `<input>`)

Reply:
Personal preference only. Didn't check if it would work 😅.